### ISSUE TITLE
Link Districts to Locations

### ISSUE THREAD

**Issue by jonmgeiger** [jonmgeiger/household-conditions#6]
Our data has a list of school districts, but we're given no information about what counties/states/areas those school districts are in. Some of them might even span multiple states. We need to find a good data set that we can join with our main data to be able to work with maps, but we're not given any off the bat. 

Some ideas: 
- Check out Department of Education to see if they have any data on school districts
- Check out the [Urban Institute report](https://www.urban.org/research/publication/mapping-student-needs-during-covid-19) to see how they got location from the district data. 
- At the Urban Institute, try [searching for data based on the tags](https://datacatalog.urban.org/search/field_tags/education-188) of the original dataset to see if there's anything useful there.

**Comment by joppaa** [comment]
District Boundaries Data: https://nces.ed.gov/programs/edge/Geographic/DistrictBoundaries

**Comment by joppaa** [comment]
District Boundaries Data uses TIGER line shapefiles - need to use this r-package to download files from the above site. 
https://www.rdocumentation.org/packages/tigris/versions/1.6

**Comment by jonmgeiger** [comment]
@joppaa Can we aim to have this done by Friday? I think that using the `tidycensus` package will be very useful for this. I'd recommend making a pull request and linking this issue to it.

**Comment by joppaa** [comment]
Yes! I will aim to get it done by Friday night! I work mornings so it will be the afternoon's focus

**Comment by jonmgeiger** [comment]
@joppaa Check out [this link](https://walker-data.com/tidycensus/articles/spatial-data.html) for using Spatial Data in the `tidycensus` package.
